easybuild/easyblocks/a/aomp.py

```diff
@@ -0,0 +1,127 @@
+"""
+Support for building and installing AOMP - AMD OpenMP compiler, implemented as
+an EasyBlock
+
+@author: Jorgen Nordmoen (University Center for Information Technology - UiO)
+"""
+
+from easybuild.easyblocks.generic.binary import Binary
+from easybuild.framework.easyconfig import CUSTOM
+from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.config import build_option
+from easybuild.tools.modules import get_software_root
+import os
+import os.path
+
+AOMP_ALL_COMPONENTS = ['roct', 'rocr', 'project', 'libdevice', 'openmp',
+                       'extras', 'pgmath', 'flang', 'flang_runtime', 'comgr',
+                       'rocminfo', 'vdi', 'hipvdi', 'ocl', 'rocdbgapi',
+                       'rocgdb', 'roctracer', 'rocprofiler']
+AOMP_DEFAULT_COMPONENTS = ['roct', 'rocr', 'project', 'libdevice', 'openmp',
+                           'extras', 'pgmath', 'flang', 'flang_runtime',
+                           'comgr', 'rocminfo']
+AOMP_X86_COMPONENTS = ['vdi', 'hipvdi', 'ocl']
+AOMP_DBG_COMPONENTS = ['rocdbgapi', 'rocgdb']
+AOMP_PROF_COMPONENTS = ['roctracer', 'rocprofiler']
+
+
+class EB_AOMP(Binary):
+    """Support for installing AOMP"""
+
+    @staticmethod
+    def extra_options():
+        extra_vars = Binary.extra_options()
+        extra_vars.update({
+            'components': [None, "AOMP components to build. Possible components: " +
+                           ', '.join(AOMP_ALL_COMPONENTS), CUSTOM],
+            'cuda_compute_capabilities': [[], "List of CUDA compute capabilities to build with", CUSTOM],
```

Comment:
This is available in the framework already and is not needed here.

Reply:
I have removed it now